packages/dina-style/src/site/collections/menu.variables

```diff
@@ -1,3 +1,5 @@
 /*******************************
     User Variable Overrides
 *******************************/
+
+@itemHoverBackground: @segmentBackground;
```

Comment:
are you sure it should always be this color? what if the menu item is not in a segment